assets/js/session/index.js

```diff
@@ -522,6 +529,18 @@ function saveNotebook(hook) {
   hook.pushEvent("save", {});
 }
 
+function captureCodeSnippet(hook) {
+  if (hook.state.focusedCellId) {
+    html2canvas(
+      document.querySelector("#cell-" + hook.state.focusedCellId)
+    ).then((canvas) => {
+      const dataURL = canvas.toDataURL("image/png");
+      const newTab = window.open("about:blank", "livebook code snippet");
```

Comment:
The second argument doesn't specify the title, so we can leave it as default. To set the title you could do:

```js
newTab.document.title = "Livebook code snippet";
```

Reply:
Done!